src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double high; //Add random parameters of Figure
+    private final double firstSide;
+    private final double secondSide;
+
+    private final Color color;
+
+    public IsoscelesTrapezoid(Color color, double high, double firstSide, double secondSide) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables for high, firstSide, and secondSide and pass them on for better code readability. This will make it easier to understand the values that are being passed to the constructor and will also make the code more maintainable.